lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -413,10 +414,17 @@ func buildCreate(ctx context.Context, d *data.Data, finder *find.Finder, vch *mo
 				c.Proxies = fromImageFetchProxy(vch.Registry.ImageFetchProxy)
 				_, _, err := c.Proxies.ProcessProxies()
 				if err != nil {
-					return nil, util.NewError(400, fmt.Sprintf("Error processing proxies: %s", err))
+					return nil, util.NewError(http.StatusBadRequest, fmt.Sprintf("Error processing proxies: %s", err))
 				}
 			}
 		}
+
+		if vch.SyslogAddr != "" {
+			c.SyslogAddr = vch.SyslogAddr.String()
+			if err := c.ProcessSyslog(); err != nil {
+				return nil, util.NewError(http.StatusBadRequest, fmt.Sprintf("Error processing syslog server address: %s", err))
+			}
+		}
```

Comment:
Why can't you write a simple unit test to verify these new code paths?

Reply:
This is part of the create API handler and we haven't figured out a pattern to do unit tests on swagger handlers yet. 
The new code path validates a field in http GET request JSON, and it's very hard to test it separately from all the other fields related to vch create params. 
We're planning to integrate this in unit tests for create API as a whole (the tracking story is: https://github.com/vmware/vic/issues/6018)